quinn-h3/src/server.rs

```diff
@@ -336,13 +690,84 @@ impl Future for RecvRequest {
     }
 }
 
+/// Send a response back to the client.
+///
+/// This struct is made available once request headers are received, when [`RecvRequest`] resolves.
+/// The application can then send the resulting [`http::Response`] with [`send_response()`]. It can
+/// be used independently with [`BodyReader`], so you can choose whether the application needs to
+/// receive the body prior to issuing a response or if a response can be issued right away.
+///
+/// The request can also be cancelled with [`cancel()`], after which the client will receive a request
```

Comment:
What's the difference between `reject()` (above) and `cancel()`?

Reply:
There is no significant difference, I guess i as influenced by the draft terminology here.

Reply:
Actually there is a strong difference: Cancellation means some data has been processed by the application, and it chose to abandon the response.